errs/error.go

```diff
@@ -0,0 +1,197 @@
+package errs
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// StatusCoder interface is used by errors that returns the HTTP response code.
+type StatusCoder interface {
+	StatusCode() int
+}
+
+// StackTracer must be by those errors that return an stack trace.
+type StackTracer interface {
+	StackTrace() errors.StackTrace
+}
+
+// Error represents the CA API errors.
+type Error struct {
+	Status int
+	Err    error
+	Msg    string
```

Comment:
Message instead of Msg

Reply:
If I change this to Message then we'd need to rename the method `Message`, implements the friendlyerror type.

Reply:
👍